src/components/application_manager/test/include/application_manager/mock_message_helper.h

```diff
@@ -160,6 +160,11 @@ class MockMessageHelper {
                mobile_apis::Result::eType(smart_objects::SmartObject& message,
                                           ApplicationConstSharedPtr app,
                                           ApplicationManager& app_mngr));
+  MOCK_METHOD3(VerifyImage,
```

Comment:
@wolfylambova22. this mock-method i've already added to develop. Search by file and will see that you duplicate. Delete it please.